pennylane/devices/default_mixed.py

```diff
@@ -23,9 +23,10 @@
 import itertools
 from string import ascii_letters as ABC
 
-import numpy as np
-from pennylane import QubitDevice, QubitStateVector, BasisState, DeviceError
-from pennylane.operation import Channel
+import pennylane.numpy as np
+import pennylane.math as qnp
+from pennylane import QubitDevice, QubitStateVector, BasisState, DeviceError, QubitDensityMatrix
+from pennylane.operation import DiagonalOperation, Channel
```

Comment:
This was added when merging in master
```suggestion
from pennylane.operation import Channel
```